recipes/scitime/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: scitime
+  version: {{ version }}
+
+source:
+  url: https://github.com/nathan-toubiana/scitime/archive/{{ version }}.tar.gz
+  sha256: 05f11512343bbaf521c6358e8321a8555d0ef4fe41a76ced9a8234d8b8bbab5b
+
+build:
```

Comment:
Can this be a noarch: python package?

Reply:
unfortunately the build fails on win for some reason - so the checks won't pass if I remove the selector

Reply:
The build failing for noarch on windows is fine. noarch only needs to build on circle

Reply:
ok I changed it then

Reply:
build is successful on circle